pagopa/pn-frontend-e2e
# PN-9309-e-2-e-destinatario-recapiti-cortesia-modifica-email

## Short description
There are some missing feature steps for the **PN-9309** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868811083/Cittadino+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added missing feature steps:
```
- And Si visualizzano correttamente i pulsanti modifica, elimina ed è possibile modificare l'email
- And Nella pagina I Tuoi Recapiti si inserisce l'email errata "provà&@gmail.com"
- Then Nella pagina I Tuoi Recapiti si visualizza correttamente il messaggio email errata
- And Nella pagina I Tuoi Recapiti si inserisce un email maggiore di 255 caratteri
- Then Nella pagina I Tuoi Recapiti si visualizza correttamente il messaggio email errata
```
- added related checks and methods


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_modificaEmailPF` and verify if the test works fine.
